src/app/public/modules/list-view-grid/list-view-grid.component.scss

```diff
@@ -0,0 +1,7 @@
+:host {
+  ::ng-deep {
+    .sky-grid-heading {
+      border-top: 0 !important;
```

Comment:
Somewhere along the line a border-top was added to the grid heading. Added this to get our visual tests to pass.

Reply:
I think this was from a toolbar fix we did recently.